src/components/graphic.jsx

```diff
@@ -12,7 +12,7 @@ function Graphic({ graphics }) {
             && graphics.map((graphic) => (
               <Grid
                 className="usa-media-block"
-                tablet={{ col: 6 }}
+                tablet={{ col: 6 }} // TODO: maybe adjust for different number of graphics?
```

Comment:
From now on can we add todos in airtable? One of the code review comments was to remove unneeded comments

Reply:
Sure.